rpc/common/moc_common_computecommon.proto

```diff
@@ -83,4 +83,26 @@ enum VirtualMachineOperation {
 	RESET = 2;
 }
 
+enum VirtualMachineRunCommandExecutionState {
+	ExecutionState_FAILED = 0;
+	ExecutionState_SUCCEEDED = 1;
+	ExecutionState_UNKNOWN = 2;
```

Comment:
I would put "unknown" as 0, because 0 is frequently the default value.

Reply:
So, if a message is coming to a server/client and the execution state field is missing, then it would automatically be mapped to "unknown" instead of "failed"